### PR TITLE
chore(query): fix ceil/floor behavior

### LINKED ISSUE #7343: The return type of `ceil` function
**Summary**

Currently, the `ceil` return type is f64.

But the definition of `ceil(x)` is **Returns the smallest integer greater than or equal to x**.

So we should consider changing the current behavior.

Btw, our doc is confusing: https://databend.rs/doc/reference/functions/numeric-functions/ceil

image

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

chore(query): fix ceil/floor behavior

Fixes #7343

### CODE REVIEW

File: docs/doc/30-reference/20-functions/20-numeric-functions/ceil.md [hunk 1]
```diff
@@ -21,17 +19,17 @@ CEIL(x)
 
 ## Return Type
 
-A Float64 data type value.
+A numeric data type value which is the same as input type.
```
Comment (xudong963) [inline docs/doc/30-reference/20-functions/20-numeric-functions/ceil.md:22]: What database is this behavior aligned to? As far as I know, mysql and postgresql return type is an integer

File: tests/suites/0_stateless/13_tpch/13_0005_q5.result [hunk 2]
```diff
@@ -1,5 +1,5 @@
 CHINA	7822103.0
```
Comment (xudong963) [inline tests/suites/0_stateless/13_tpch/13_0005_q5.result:1]: Seems the value of CHINA sometimes is `7822102.999`, so if using `floor`, the result is 7822102.0

File: docs/doc/30-reference/20-functions/20-numeric-functions/ceil.md (hunk 1 above)
Comment (sundy-li) [reply docs/doc/30-reference/20-functions/20-numeric-functions/ceil.md:22]: I referred to duckdb and clickhouse.

PG returns numeric type too.  

https://www.db-fiddle.com/f/oAKgKRCktppWXb31Zw3Sws/0

File: tests/suites/0_stateless/13_tpch/13_0005_q5.result (hunk 2 above)
Comment (sundy-li) [reply tests/suites/0_stateless/13_tpch/13_0005_q5.result:1]: -CHINA	7822103.0
+CHINA	7822104.0

I'd like to ignore the last number...

Comment (xudong963) [reply tests/suites/0_stateless/13_tpch/13_0005_q5.result:1]: Strange, ceil(7822103.0) -> 7822103.0, why occur 7822104.0 🤔

Comment (sundy-li) [reply tests/suites/0_stateless/13_tpch/13_0005_q5.result]: sometimes it'll be 7822103.x.

**Review by xudong963 — APPROVED** [review]

**Review by b41sh — APPROVED** [review]
LGTM

**Pull request merged**